Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1080,8 +1087,8 @@
     <RegEx find="peškir(?=i(ma)?\b)" replaceWith="ručnic" />
     <RegEx find="Peškir(?=i(ma)?\b)" replaceWith="Ručnic" />
     <RegEx find="([pP])eva" replaceWith="$1jeva" />
-    <RegEx find="pijac([aeiuo])" replaceWith="tržnic$1" />
-    <RegEx find="Pijac([aeiuo])" replaceWith="Tržnic$1" />
+    <RegEx find="\bpijac" replaceWith="tržnic" />
+    <RegEx find="Pijac" replaceWith="Tržnic" />
```

Comment:
there's no need in this case, since it's first letter, so nothing is before it.
I'll have to remove all similar cases in the future, as well, like names.